self-api/src/main/java/com/selfxdsd/api/ProjectsPaged.java

```diff
@@ -0,0 +1,12 @@
+package com.selfxdsd.api;
+
+import com.selfxdsd.api.storage.Paged;
+
+/**
+ * Paginated version of {@link Projects}.
+ * @author criske
+ * @version $Id$
+ * @since 0.0.13
+ */
+public interface ProjectsPaged extends Projects, Paged {
```

Comment:
@criske Do we really need this new interface? I would make it like this: ``Projects extends Paged<Project>``. And then add an abstract class called ``PagedProjects``, which would be extended by ``StoredProjects``. Wouldn't that be possible?

Reply:
@amihaiemil Sure we could do that. With my aproach we could list projects either paged either all at once. But I guess, in hindsight, having an endpoint to all projects is useless in our case :D . I'll change the design and see if it works.